blueprint/api.apib

```diff
@@ -279,6 +279,165 @@ You can list all comments or retrieve individual comments, as well as create and
 
     + Attributes (Unprocessable Entity Response)
 
+# Group Donation Goals
+
+This resource describes one of any number of donation goals a project might have. A project cannot create a donation goal until they have started accepting payment with Stripe.
+
+## Donation Goals [/donation-goals]
+
+### Create a donation goal [POST]
+
+To create a donation goal the user must be a site admin or the owner of the referenced project.
+
++ Parameters
+
+    + id (number) - The `id` of a donation goal.
+
++ Request
+
+    + Headers
+
+            Accept: application/vnd.api+json
+            Authorization: Bearer {token}
+
++ Response 201 (application/vnd.api+json; charset=utf-8)
+
+    + Attributes (Donation Goal Create Request)
```

Comment:
Here you're defining the response as a request. Look at some other create request examples to see where this might be defined, as well as how the request should look.